Themis/Views/Assessment/AssessmentSubmissionLoaderView.swift

```diff
@@ -25,6 +25,7 @@ struct AssessmentSubmissionLoaderView: View {
             await avm.getSubmission(id: submissionID)
             if let pId = avm.submission?.participation.id {
                 await cvm.initFileTree(participationId: pId)
+                await cvm.loadInlineHighlight(assessmentResult: avm.assessmentResult, participationId: pId)
```

Comment:
Move this into assessment view on appear as I already move `cvm.initFileTree` there